docs/guides/admin/docs/configuration/security.jwt.md

```diff
@@ -102,79 +102,44 @@ An external application wants to let a user use Opencast Studio to record and up
 Spring Security Configuration
 -----------------------------
 
-In order to active JWT-based authentication and authorization, you will need to uncomment and adapt the following
-sections found in `etc/security/mh_default_org.xml`. Some of the options are configured with the
-[Spring Expression Language (SpEL)](https://docs.spring.io/spring-framework/docs/3.0.x/reference/expressions.html).
+In order to active JWT-based authentication and authorization, you will need to adjust your security config (e.g. `etc/security/mh_default_org.xml`).
```

Comment:
I suppose this should say
```suggestion
In order to activate JWT-based authentication and authorization, you will need to adjust your security config (e.g. `etc/security/mh_default_org.xml`).
```

That typo was there before, but since you're changing this part anyway, maybe also fix that.